base_export_async/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_delay_export,delay.export.user,model_delay_export,,1,1,1,1
```

Comment:
Since delay.model only has only model methods, is this ACL required?

Reply:
Yes to avoid Warning in the log (travis doesn't like that)